src/contracts/core/DelegationManager.sol

```diff
@@ -535,137 +572,151 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
+                uint256 nonNormalizedShares = SlashingAccountingUtils.denormalize({
+                    shares: shares,
+                    scalingFactor: slasher.shareScalingFactor(currentOperator, withdrawal.strategies[i])
+                });
+
+                // TODO DONE?: figure out how to award correct number of shares back when a stale withdrawal is completed
+                // Award shares back in StrategyManager/EigenPodManager. If withdrawer is delegated, increase the shares delegated to the operator
+                // When awarding podOwnerShares in EigenPodManager, we need to be sure to only give them back to the original podOwner.
+                // Other strategy shares can + will be awarded to the withdrawer.
+                if (withdrawal.strategies[i] == beaconChainETHStrategy) {
+                    // Update shares amount depending upon the returned value.
+                    // The return value will be lower than the input value in the case where the staker has an existing share deficit
+                    nonNormalizedShares = eigenPodManager.addShares({
+                        podOwner: withdrawal.staker,
+                        shares: nonNormalizedShares
+                    });
+                } else {
+                    strategyManager.addShares(msg.sender, tokens[i], withdrawal.strategies[i], nonNormalizedShares);
+                }
+
+                // Similar to `isDelegated` logic
+                if (currentOperator != address(0)) {
+                    _increaseOperatorShares({
+                        operator: currentOperator,
+                        // the 'staker' here is the address receiving new shares
+                        staker: withdrawal.staker,
+                        strategy: withdrawal.strategies[i],
+                        nonNormalizedShares: nonNormalizedShares
+                    });
                 }
-                unchecked { ++i; }
             }
+            unchecked { ++i; }
         }
-
+        
         emit WithdrawalCompleted(withdrawalRoot);
     }
 
-    // @notice Increases `operator`s delegated shares in `strategy` by `shares` and emits an `OperatorSharesIncreased` event
-    function _increaseOperatorShares(address operator, address staker, IStrategy strategy, uint256 shares) internal {
-        operatorShares[operator][strategy] += shares;
-        emit OperatorSharesIncreased(operator, staker, strategy, shares);
+    // @notice Increases `operator`s delegated nonNormalizedShares in `strategy` by `nonNormalizedShares` and emits an `OperatorSharesIncreased` event
+    function _increaseOperatorShares(address operator, address staker, IStrategy strategy, uint256 nonNormalizedShares) internal {
+        nonNormalizedOperatorShares[operator][strategy] += nonNormalizedShares;
+        emit OperatorSharesIncreased(operator, staker, strategy, nonNormalizedShares);
```

Comment:
Generally, I am not a fan of changing the existing "shares" naming to `nonNormalizedShares`.

IMO we keep the existing "shares" concept as-is, and use `normalizedShares` after we do whatever conversion normalization entails. Changing these interfaces and variable names though both:
* breaks more things
* requires more thinking